src/store/features/display/display-selectors.ts

```diff
@@ -9,6 +12,12 @@ export const selectIsFieldVisible =
   (state: RootState): boolean =>
     state.display.visibleFields.includes(field);
 
+export const selectSortBy = (state: RootState): TaskDisplayField =>
+  state.display.sortBy;
+
+export const selectSortDirection = (state: RootState): 'asc' | 'desc' =>
```

Comment:
Sort direction could be extracted out as a type since it's repeated in a few places